impl-endpoints/cloudflare-go/run_endpoint.sh

```diff
@@ -9,12 +9,12 @@ sh /setup-routes.sh
 
 if [ "$ROLE" = "client" ]; then
     echo "Running Cloudflare-Go client."
-    echo "Client params: $CLIENT_PARAMS"
+    # echo "Client params: $CLIENT_PARAMS"
```

Comment:
Let's just delete this if it's no longer needed. (Same for the commented out line below.)